src/components/Form/TextInput.tsx

```diff
@@ -83,13 +91,19 @@ class BaseTextInput extends Component<TextInputProps> {
         ) : (
           <Paragraph {...labelTextProps}>{labelText}</Paragraph>
         )}
-        <HtmlDiv {...inputContainerProps}>
-          <HtmlInput
-            {...passProps}
-            className={classnames(inputBaseClassName, inputClassName)}
-            type="text"
-          />
-          {children}
+        <HtmlDiv className={stateTextClassName}>
+          <HtmlDiv {...inputContainerProps}>
+            <HtmlInput
+              {...passProps}
+              className={classnames(inputBaseClassName, inputClassName)}
+              type="text"
+              aria-describedby={generatedId}
+            />
+            {children}
+          </HtmlDiv>
+          <HtmlLabel className={stateTextLabelClassName} id={generatedId}>
```

Comment:
Is it intended to apply the id prop to label? Shouldn't the id be added to the actual input element or does aria change the way these are handled? How does this work overall if I give the TextInput component an id? Can I expect that id to be present in the underlying input and if not, could it have undesired side effects?

Reply:
It is by mistake, label should have it's own id and not the input's id-prop. Will be fixed.

Reply:
This should work too, unless one gives the same id to multiple inputs?

Reply:
Yes, that is true.

But the `id` [must be unique in the whole document](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) by the definition.
It was possible to give the `id` attribute for the TextInput even before this change, so user could use it wrong already.

If not supporting like this, I couldn't find a way to have working snapshots as `idGenerator` generates new one for every test run. But I'm open for suggestion for alternative working solution.

Reply:
It's fine, my latest comment should be ignored.